tsup.config.ts

```diff
@@ -0,0 +1,37 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
why exactly do we need this? i thought the CLI is ESM only already, and the main benefit of tsup was dual build packages?

Reply:
This is for node+npm users, until we make node just wrap around the bundles, because I wouldn't wanna block a v1 release on that (unless you disagree in which case ik what to work on after upgrade)

Reply:
but you dont need bun to run a script bundled by bun, no? my point is you are introducing new dependency which i dont see a need for right now. its a minor issue, i dont want to block anything because of it, just trying to understand the reasoning behind it